packages/components/src/index.js

```diff
@@ -8,3 +8,4 @@ export { default as Ribbon } from './ribbon';
 export { default as RootChild } from './root-child';
 export { default as ScreenReaderText } from './screen-reader-text';
 export { default as Suggestions } from './suggestions';
+export { default as WhatsNewGuide } from './whats-new';
```

Comment:
🤔 

I expected to have this in its own package, to be honest. The components in `@automattic/components` are usually simple React components used for composing the UI of Calypso. They don't connect to Redux, nor they perform HTTP requests, record analytics / tracks events, or trigger any other side effects.